core/domain/exp_services_test.py

```diff
@@ -2887,10 +2887,12 @@ def test_accept_suggestion_valid_suggestion(self):
             with self.swap(exp_services, 'update_exploration',
                            self._check_commit_message):
                 with self.swap(user_services, 'get_human_readable_user_ids',
-                               self. _get_username_):
-                    exp_services.accept_suggestion(
-                        'user_id', 'change_list', self.THREAD_ID1, self.EXP_ID1,
-                        self.COMMIT_MESSAGE)
+                               self._get_username_):
```

Comment:
I just noticed this; why is there a _ at the end of _get_username_?

Also, seems a bit of a weird thing to swap. Why not just register the user at the outset?